fluentd/Dockerfile.centos7

```diff
@@ -40,6 +40,7 @@ RUN mkdir -p ${HOME} && mkdir -p /etc/fluent/plugin && \
      'fluent-plugin-systemd:<0.1.0' \
       fluent-plugin-viaq_data_model \
      'fluent-plugin-remote-syslog:1.1' \
+      fluent-plugin-viaq_docker_audit_log_parser \
```

Comment:
If we vendor this plugin, will this  be true?  Wont we need to source it from a local dir? possibly build it too?

Reply:
The ruby .rb file should be added to Dockerfile.centos7 and Dockerfile, and the dockerfile should copy the .rb file into /etc/fluent/plugin/